packages/commonwealth/client/scripts/views/pages/discussions/DiscussionsPage.tsx

```diff
@@ -48,6 +50,9 @@ const DiscussionsPage = ({ topicName }: DiscussionsPageProps) => {
     dateRange: searchParams.get('dateRange') as ThreadTimelineFilterTypes,
   });
 
+  const onArchivePage =
+    location.pathname === `/${app.activeChainId()}/archived`;
```

Comment:
should we do `.toLowerCase()`, `/Archived` / `/arChivEd` or any other lettercase combo will break it

Reply:
Testing in the browser, I found that the pathname is downcased regardless of the casing you enter.